scripts/clones_analysis/clones_features_builder.py

```diff
@@ -0,0 +1,50 @@
+import networkx as nx
+import pandas as pd
+
+
+def add_number_of_clones_features(methods_df: pd.DataFrame, graph: nx.Graph = None):
+    methods_df['n_clones'] = methods_df['method_id'].apply(lambda id: number_of_clones(id, graph))
```

Comment:
Please, create an Enum class with these keys. You can potentially make a mistake when you copy the string. In addition, if you need to change the name, it must be changed in all unrelated places.